### DISCUSSION TITLE
Midi Support

### DISCUSSION THREAD

**Discussion by KeenVox** [frntc/Sidekick64#15] [Q&A] [answered]
Hi, 
how does the midi support/sequencer work? I don't seem to find anything except in the feature list.

**Comment by frntc** [comment, marked as answer]
The Datel and Sequential MIDI interfaces receive data via the IO-address range which is available at the expansion port and thus to Sidekick. It simple reads commands to those MIDI devices and emulates a GM device using TSF. Note that the Sidekick does not support more elaborate things like IRQs which these interfaces do. If you're looking for a complete emulation, you might want to have a look at my SIDKick project.
